### PR TITLE
Get ceph-dashboard's public addresses from the mon

### PR DESCRIPTION
**Pull request by ChrisMacNaughton**
At some point, Juju stopped returning a list of units for subordinate charms, so this test now fails to find the address for the dashboard units

### CODE REVIEW

**Comment by ajkavanagh** [comment]
@ChrisMacNaughton as per the victoria change, is there an associated review that verifies this change please?

**Comment by ChrisMacNaughton** [comment]
https://review.opendev.org/c/openstack/charm-ceph-dashboard/+/831510 is verifying this for Ussuri / Ceph Octopus and Pacific; the Victoria change is a bit awkward, as I only actually included it to be able to backport this change to Ussuri :-/

**Comment by ChrisMacNaughton** [comment]
https://review.opendev.org/c/openstack/charm-ceph-dashboard/+/844512 is now being used to validate this version, as the previous link is being used to validate Victoria

**Review by freyes — APPROVED** [review]
LGTM. Waiting for another CI run from the associated ceph-dashboard change before merging

**Pull request merged**
